nms/v1_14R1/src/main/java/net/dzikoysk/funnyguilds/nms/v1_14R1/playerlist/V1_14R1PlayerList.java

```diff
@@ -16,7 +16,7 @@
 import net.minecraft.server.v1_14_R1.PacketPlayOutPlayerInfo;
 import net.minecraft.server.v1_14_R1.PacketPlayOutPlayerInfo.EnumPlayerInfoAction;
 import net.minecraft.server.v1_14_R1.PacketPlayOutPlayerListHeaderFooter;
-import org.apache.commons.lang.StringUtils;
+import org.bukkit.craftbukkit.libs.org.apache.commons.lang3.StringUtils;
```

Comment:
Dlaczego z libów craftbukkita? Probably a typo.

Reply:
Because in nms modules we jedynie mamy apache commons 3 from bukkit, które bukkit relocates. Otherwise we would musieć use apache commons 2. I don't sądzę, it's typo.

Reply:
In NMS modules - we use imports directly from the server engine, so if CraftBukkit relocates lang3 - we can use that. We already use the internal NMS code for that specific version, so usage of such relocated CraftBukkit imports will be totally safe